packages/sample/src/main.ts

```diff
@@ -74,67 +109,146 @@ program
+        "Read the current battery Voltage. Specify --continuous to run continuously",
     )
     .action(async (options) => {
+        let isContinuous = options.continuous !== undefined;
+        let hubs = await openConnectedExpansionHubs();
+        let hub = hubs[0];
+
         runOnSigint(() => {
             hub.close();
         });
 
+        await batteryVoltage(hub, isContinuous);
+        hub.close();
+    });
+
+batteryCommand
+    .command("current")
+    .option("--continuous", "Run continuously")
+    .description("Read the battery current. Specify --continuous to run continuously")
+    .action(async (options) => {
         let isContinuous = options.continuous !== undefined;
         let hubs = await openConnectedExpansionHubs();
         let hub = hubs[0];
-        await voltageRail(hub, isContinuous);
+
+        runOnSigint(() => {
+            hub.close();
+        });
+
+        await batteryCurrent(hub, isContinuous);
         hub.close();
     });
 
 program
-    .command("battery")
+    .command("i2c-current")
     .option("--continuous", "Run continuously")
     .description(
-        "Read the current battery Voltage. Specify --continuous to run continuously",
+        "Read the I2C sub-system current. Specify --continuous to run continuously",
     )
     .action(async (options) => {
+        let isContinuous = options.continuous !== undefined;
+        let hubs = await openConnectedExpansionHubs();
+        let hub = hubs[0];
+
         runOnSigint(() => {
             hub.close();
         });
 
+        await i2cCurrent(hub, isContinuous);
+        hub.close();
+    });
+
+program
+    .command("digital-current")
+    .option("--continuous", "Run continuously")
+    .description("Read the digital bus current. Specify --continuous to run continuously")
+    .action(async (options) => {
+        let hubs = await openConnectedExpansionHubs();
+        let hub = hubs[0];
         let isContinuous = options.continuous !== undefined;
+
+        runOnSigint(() => {
+            hub.close();
+        });
+
+        await digitalBusCurrent(hub, isContinuous);
+        hub.close();
+    });
+
+program
+    .command("servo-current")
+    .option("--continuous", "Run continuously")
+    .description(
+        "Read the total current through all servos. Specify --continuous to run continuously",
+    )
+    .action(async (options) => {
         let hubs = await openConnectedExpansionHubs();
         let hub = hubs[0];
-        await battery(hub, isContinuous);
+        let isContinuous = options.continuous !== undefined;
+
+        runOnSigint(() => {
+            hub.close();
+        });
+
+        await servoCurrent(hub, isContinuous);
         hub.close();
     });
 
 program
     .command("servo <channel> <pulseWidth> [frameWidth]")
     .description("Run a servo with pulse width and optional frame width")
     .action(async (channel, pulseWidth, frameWidth) => {
-        runOnSigint(async () => {
```

Comment:
I moved these because the hub wasn't semantically declared before this. It's very unlikely to cause any issues (unless we get a sigint before it finishes opening the hub).